qemu/tests/balloon_boot_in_pause.py

```diff
@@ -19,11 +19,13 @@ def __init__(self, test, params, env):
         super(BallooningTestPause, self).__init__(test, params, env)
 
         self.vm = env.get_vm(params["main_vm"])
+        # ori_mem/gmem is the original memory
+        # pre_mem/gmem is the memory value before balloon test
         self.ori_mem = self.get_vm_mem(self.vm)
         if self.get_ballooned_memory() != self.ori_mem:
             self.balloon_memory(self.ori_mem)
-        self.old_mmem = self.ori_mem
-        self.old_gmem = None
+        self.pre_mmem = self.ori_mem
+        self.pre_gmem = None
```

Comment:
L68 ori_mem should be replaced with pre_*.

Reply:
Modify.